src/form-engine.component.tsx

```diff
@@ -255,11 +254,6 @@ const FormEngine: React.FC<FormProps> = ({
               <LoadingIcon />
             ) : (
               <div className={styles.formEngineContainer}>
-                {isLoadingFormDependencies && (
-                  <div className={styles.loader}>
-                    <LinearLoader />
-                  </div>
-                )}
```

Comment:
@denniskigen this renders the `isLoadingFormDependencies` state redundant. We either remove or use it. I think we still need some kind of loader for form dependencies. I've no objection with moving this to the standard loader.

Reply:
Sure. So I've incorporated `LinearLoader` into the `FormEngine` component in my latest commits. I've also replaced the `UiSelectExtended` component's loader with a [DropdownSkeleton](https://react.carbondesignsystem.com/?path=/story/components-dropdown--skeleton).